tests/apollo/util/bft.py

```diff
@@ -394,13 +397,27 @@ def start_replica(self, replica_id):
         with log.start_action(action_type="start_replica"):
             stdout_file = None
             stderr_file = None
+            storage_type = os.environ.get("STORAGE_TYPE")
+            tests_names = [m for m in sys.modules.keys() if m.startswith("test_")]
```

Comment:
@oded282 you are maybe right about the env variable. I don't like to have STORAGE_TYPE propagate down here.
Let's try your suggestion with the TEST_NAME env variable.

Reply:
@teoparvanov , @oded282 , please notice that`make test` does not trigger either cmake generation or compilation. The reason is that tests are not supposed to change the pre-built product.
Please keep this behavior.

Reply:
ok